javaslang-benchmark/src/test/java/javaslang/control/LazyBenchmark.java

```diff
@@ -0,0 +1,83 @@
+package javaslang.control;
+
+import javaslang.JmhRunner;
+import javaslang.Lazy;
+import javaslang.collection.Array;
+import javaslang.collection.Iterator;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
+
+import static javaslang.JmhRunner.Includes.JAVA;
+import static javaslang.JmhRunner.Includes.JAVASLANG;
+
+public class LazyBenchmark {
+    static final Array<Class<?>> CLASSES = Array.of(
+            Get.class
+    );
+
+    @Test
+    public void testAsserts() { JmhRunner.runDebugWithAsserts(CLASSES); }
+
+    public static void main(String... args) {
+        JmhRunner.runDebugWithAsserts(CLASSES, JAVA, JAVASLANG);
+        JmhRunner.runSlowNoAsserts(CLASSES, JAVA, JAVASLANG);
+    }
+
+    @State(Scope.Benchmark)
+    public static class Base {
+        final int SIZE = 10;
+
+        Integer[] EAGERS;
+        javaslang.Lazy<Integer>[] INITED_LAZIES;
+
+        @Setup
+        @SuppressWarnings({ "unchecked", "rawtypes" })
+        public void setup() {
+            EAGERS = Iterator.range(0, SIZE).toJavaArray(Integer.class);
+            INITED_LAZIES = Iterator.of(EAGERS).map(i -> {
+                final Lazy<Integer> lazy = Lazy.of(() -> i);
+                lazy.get();
+                return lazy;
+            }).toJavaList().toArray(new Lazy[0]);
```

Comment:
I agree. It will be a little less ugly if we add `T[] Value.toJavaArray(T[])` - the only safe solution :-/
See #1582